debian/libibverbs-dev.install

```diff
@@ -13,6 +13,7 @@ usr/include/infiniband/verbs_api.h
 usr/lib/*/libibverbs*.so
 usr/lib/*/libmlx4.so
 usr/lib/*/libmlx5.so
+usr/lib/*/pkgconfig/libibverbs.pc
```

Comment:
Where are the pkgconfigs for libmlx5 and 4?

Reply:
providers/mlx5 and providers/mlx4, they are intended as application linkable libraries and as provider plugins, eg dpdk links to them.